entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo $DB_CONFIG > /etc/sns/db.json
```

Comment:
I think this will overwrite existing config, even if `DB_CONFIG` is not specified.

Also config location can be different (set via DB_PATH)